tests/ignite/metrics/test_accuracy.py

```diff
@@ -366,18 +366,23 @@ def _test(n_epochs, metric_device):
         s = 16
         n_classes = 10
 
+        torch.manual_seed(12 + rank)
+
         offset = n_iters * s
```

Comment:
Same for `offset`, let's rename it or even remove it

Reply:
I change `s` to `batch_size` and eliminate `offset`